docs/001_develop/03_client-capabilities/009_presentation/icon.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: 'Icon'
+sidebar_label: 'Icon'
+id: client-presentation-icon
+keywords: [presentation, icon, image]
+tags:
+- presentation
+- icon
+- image
+sidebar_position: 9
+---
+
+import IconDemo from './examples/icon.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Icon
+
+Use cases:
+* Displaying a specified icon.
+* An addition to presenting a listbox-option
+* cell-renderers and many others
+
+:::note
+You can use any free publicly-available icon from fontawesome. These are available via the links below:
```

Comment:
I think adding a very brief explanation about paying for fontawesome pro could be good here too. Need to double check but I basically think if you get a licence for your app then you'll be able to use all those icons via this component too